S181 is not connected (#1716)
pi-base/data: properties/P000036.md

#### Meta-properties

- $X$ satisfies this property iff its Kolmogorov quotient $\text{Kol}(X)$ does.

> - An arbitrary product satisfies this property iff each of its factors does.

Reviewer: This is not true (needs to be non-empty).

> - An arbitrary product of nonempty spaces satisfies this property iff each of its factors does.

Reviewer 2: ```suggestion
- A nonempty arbitrary product $\prod_i X_i$ satisfies this property iff each $X_i$ does.
```

Reviewer: Even though I said I did not like this, you decided to change it to this anyway.

Author: Yeah, using symbols is inconsistent with how we write metaproperties usually.

Reviewer 2: @Moniker1998 nothing is decided.  I gave that suggestion to move things along, and was going to and will get you to make the final approval.  Personally I would have preferred (3.1) and (3.2).  Let me continue in the main thread.
